src/app/components/MediaLoader/index.stories.tsx

```diff
@@ -26,18 +28,36 @@ const Component = ({ service, pageType, blocks }: Props) => (
     pathname=""
     service={service}
   >
-    <MediaLoaderComponent blocks={blocks} />
+    <MediaLoaderComponent
+      blocks={blocks}
+      forcePlaceholderMode={placeholderMode}
+    />
   </RequestContextProvider>
 );
 
 export default {
   title: 'Components/MediaLoader',
   Component,
+  argTypes: {
+    placeholderMode: {
+      options: [Mode.SHOW_SUSTAINABILITY_MSG, Mode.DEFAULT],
```

Comment:
Do we want to add the `NO_JS` option here too? (Because it would actually be good to see what it would look like in Storybook which needs JS to be enabled 🤯 )